src/en/space-station-14/departments/medical/proposals/brigmedic.md

```diff
@@ -0,0 +1,55 @@
+
+## Fixing the problems
+
+> ### This is the same role except with security access to boot with none of the responsibilities of a medic, and it's primary use seemed to be security with a hardsuit, not an actual medic.
+
+We should start off by changing the mindset of brigmedic to focus more on medic instead of brig; it should be brig**medic**, not **brig**medic. Brigmedic should be removed from the security department, and be considered a member of the medical department instead. The brigmedic would still have access to the brig and security channels, similar to the civillian department's lawyer, but just like the lawyer is not a true part of security (i.e., they should not have lethals or cuffs outside of emergencies).
+
+I additionally propose that we treat "brigmedicloosing" as similar to "wardenloosing." The brigmedic *should* always be in the brig, as their title says they should; they are not secmedics. Command should be encouraged to demote brigmedics who abandon their role by leaving the brig. The sole exception to this should be when the brigmedic is instead fufilling the role of a combat medic, in which case they should be in the backlines of sec providing first aid until the officers are well enough to be brought back to the frontlines or a paramedic can retrieve them to bring to medical. We should heavily discourage brigmedics from engaging in combat except as a last resort - they should be filling the same role as a Nuclear Agent does in NukeOps for their team in the combat phase.
+
+Outside of the emergency scenario, the brigmedic's primary role would be patching up injured members of security or prisoners when they are within the brig. Some maps already have a dedicated brigmedic area to perform this duty in. As pointed out in PR discussions, prisoners do get hurt, secuirty officers can also get hurt in the process, and an injured officer bringing a prisoner and themselves to medbay creates risk of the officer critting before arrival. The brigmedic removes that risk.
+
+The brigmedic should have an extra responsibilty that normal medics do not get: ensuring the medical wellbeing of prisoners in the event of abusive or negligent security. This creates a roleplay oppourtunity akin to the lawyer fighting for the legal rights of the crew. When a bad secuirty officer wants to harm a prisoner, the brigmedic can be there to advocate for the physical and psycological safety of the prisoner (or at least repair the damage quickly if the officer does it anyways).
+
+> ### We're not readding the free hardsuit role.
+
+The brigmedic's hardsuit should be stored with the security hardsuits. Through door access controls, the brigmedic would therefore only have access to their hardsuit duing the emergency scenario, when sec lets them get the hardsuit to provide medical aid in combat. The locker containing the hardsuit should have brigmedic access to prevent a cadet from accidentally taking the hardsuit dedicated for medical use.
```

Comment:
Very good idea. Giving them their free hardsuit roundstart wasn't a good idea to begin with.